packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/components.types.ts

```diff
@@ -27,6 +27,7 @@ import {
 } from '../quick-strategy.types';
 import { TDataUniqInput } from './data/data-uniq-input-obj';
 import { TCommonInputsProperties } from './data/common-input-properties';
+import { bool } from 'prop-types';
```

Comment:
where is this used?

Reply:
sorry, that was an auto import. Removed it.